bazel: ip missing error in bazel-base container on master

* fetch latest master
* `cd experimental/bazel-base`
* `docker-compose build`
* `docker-compose run magma-builder bash`

```bash
t_uplink_gw/test.log)
INFO: Elapsed time: 101.730s, Critical Path: 13.39s
INFO: 2805 processes: 2343 remote cache hit, 446 internal, 2 local, 14 processwrapper-sandbox.
INFO: Build completed, 1 test FAILED, 2805 total actions
//:check_starlark_format                                        (cached) PASSED in 0.0s
//lte/gateway/c/li_agent/src/test:pdu_generator_test            (cached) PASSED in 0.0s
//lte/gateway/c/sctpd/src/test:event_handler_test               (cached) PASSED in 0.0s
//lte/gateway/c/sctpd/src/test:sctp_desc_test                   (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:charging_grant_test        (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:local_enforcer_test        (cached) PASSED in 0.2s
//lte/gateway/c/session_manager/test:local_enforcer_wallet_exhaust_test (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:metering_reporter_test     (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:polling_pipelined_test     (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:proxy_responder_handler_test (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:session_credit_test        (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:session_manager_handler_test (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:session_state_test         (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:session_store_test         (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:set_session_manager_handler_test (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:store_client_test          (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:stored_state_test          (cached) PASSED in 0.1s
//lte/gateway/c/session_manager/test:upf_node_state_test        (cached) PASSED in 0.1s
//lte/gateway/c/session_manager/test:usage_monitor_test         (cached) PASSED in 0.0s
//orc8r/gateway/c/common/config/test:mconfig_loader_test        (cached) PASSED in 0.0s
//orc8r/gateway/c/common/config/test:yaml_utils_test            (cached) PASSED in 0.0s
//orc8r/gateway/c/common/service303/test:magma_service_test     (cached) PASSED in 0.1s
//orc8r/gateway/c/common/service303/test:metrics_test           (cached) PASSED in 0.1s
//orc8r/gateway/c/common/service303/test:service303_test        (cached) PASSED in 0.0s
//orc8r/gateway/c/common/service_registry/test:service_registry_test (cached) PASSED in 0.0s
//src/go/agwd/config:config_test                                (cached) PASSED in 0.0s
//src/go/agwd/config/internal/grpcutil:grpcutil_test            (cached) PASSED in 0.0s
//src/go/agwd/server:server_test                                (cached) PASSED in 0.0s
//src/go/capture:capture_test                                   (cached) PASSED in 0.0s
//src/go/crash:crash_test                                       (cached) PASSED in 0.0s
//src/go/crash/sentry:sentry_test                               (cached) PASSED in 0.0s
//src/go/log:log_test                                           (cached) PASSED in 0.0s
//src/go/log/zap:zap_test                                       (cached) PASSED in 0.0s
//src/go/service:service_test                                   (cached) PASSED in 0.0s
//src/go/service/capture:capture_test                           (cached) PASSED in 0.0s
//src/go/service/config:config_test                             (cached) PASSED in 0.0s
//src/go/service/pipelined:pipelined_test                       (cached) PASSED in 0.1s
//src/go/service/sctpd:sctpd_test                               (cached) PASSED in 0.0s
//lte/gateway/c/session_manager/test:cloud_reporter_test                 PASSED in 0.5s
//lte/gateway/c/session_manager/test:sessiond_integ_test                 PASSED in 1.5s
//lte/gateway/python/magma/mobilityd/tests:ip_allocator_dhcp_mac_addr_test PASSED in 1.2s
//lte/gateway/python/magma/mobilityd/tests:ip_allocator_tests            PASSED in 12.7s
//lte/gateway/python/magma/mobilityd/tests:test_ipv6_allocator           PASSED in 2.1s
//lte/gateway/python/magma/mobilityd/tests:test_multi_apn_ip_alloc       PASSED in 2.6s
//lte/gateway/python/magma/mobilityd/tests:test_static_ip_alloc          PASSED in 8.3s
//lte/gateway/python/magma/mobilityd/tests:test_static_ipv6_alloc        PASSED in 5.2s
//lte/gateway/python/magma/mobilityd/tests:test_uplink_gw                FAILED in 1.4s
  /tmp/bazel/execroot/__main__/bazel-out/k8-fastbuild/testlogs/lte/gateway/python/magma/mobilityd/tests/test_uplink_gw/test.log

Executed 9 out of 47 tests: 46 tests pass and 1 fails locally.
There were tests whose specified size is too big. Use the --test_verbose_timeout
INFO: Build completed, 1 test FAILED, 2805 total actions
root@20be26a158a4:/magma# cat /tmp/bazel/execroot/__main__/bazel-out/k8-fastbuild/testlogs/lte/gateway/python/magma/mobilityd/tests/test_uplink_gw/test.log
exec ${PAGER:-/usr/bin/less} "$0" || exit 1
Executing tests from //lte/gateway/python/magma/mobilityd/tests:test_uplink_gw
-----------------------------------------------------------------------------
============================= test session starts ==============================
platform linux -- Python 3.8.10, pytest-6.2.5, py-1.11.0, pluggy-1.0.0 -- /usr/bin/python3.8
cachedir: .pytest_cache
rootdir: /tmp/bazel/sandbox/processwrapper-sandbox/2296/execroot/__main__/bazel-out/k8-fastbuild/bin/lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.runfiles/__main__
plugins: cov-3.0.0
collecting ... collected 8 items

lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py::DefGwTest::test_gw_ip_for_DHCP PASSED [ 12%]
lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py::DefGwTest::test_gw_ip_for_Ip_pool FAILED [ 25%]
lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py::DefGwTest::test_vlan_gw_info PASSED [ 37%]
lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py::DefGwTest::test_vlan_gw_info_list PASSED [ 50%]
lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py::DefGwTest::test_vlan_gw_info_none PASSED [ 62%]
lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py::DefGwTestIpv6::test_gw_ip_for_DHCP PASSED [ 75%]
lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py::DefGwTestIpv6::test_gw_ip_for_Ip_pool PASSED [ 87%]
lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py::DefGwTestIpv6::test_gw_ip_for_Ip_pool_v6 PASSED [100%]

=================================== FAILURES ===================================
_______________________ DefGwTest.test_gw_ip_for_Ip_pool _______________________

self = <test_uplink_gw.DefGwTest testMethod=test_gw_ip_for_Ip_pool>

    def test_gw_ip_for_Ip_pool(self):
        self.dhcp_gw_info.read_default_gw()

        def_gw_cmd = "ip route show |grep default| awk '{print $3}'"
        p = subprocess.Popen(
            [def_gw_cmd],
            stdout=subprocess.PIPE,
            shell=True,
        )
        def_ip = p.stdout.read().decode("utf-8").strip()

>       self.assertEqual(self.dhcp_gw_info.get_gw_ip(), str(def_ip))
E       AssertionError: '172.20.0.1' != ''
E       - 172.20.0.1
E       +

lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py:70: AssertionError
----------------------------- Captured stderr call -----------------------------
/bin/sh: 1: ip: not found
=========================== short test summary info ============================
FAILED lte/gateway/python/magma/mobilityd/tests/test_uplink_gw.py::DefGwTest::test_gw_ip_for_Ip_pool
========================= 1 failed, 7 passed in 0.26s ==========================
root@20be26a158a4:/magma#
```

--- a/experimental/bazel-base/Dockerfile
+++ b/experimental/bazel-base/Dockerfile
@@ -18,6 +18,8 @@ RUN apt-get update && \
         git \
         gnupg2 \
         g++ \
+        # dependency of mobilityd (tests)
+        iproute2 \
         # dependency of @sentry_native//:sentry
         libcurl4-openssl-dev \
         # dependency of sctpd

